src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 3.3.0
 -----
 
+ * Added `kernel.project_dir` and `Kernel::getProjectDir()`
+ * Deprecated `kernel.root_dir` and `Kernel::getRootDir()`
```

Comment:
well.. deprecation !== crashing :) then again migrating it is a burden.. so i see we care a bit more here.

Adding `@deprecated` sounds like a good idea :+1: